middleware/classes.py

```diff
@@ -150,3 +154,52 @@ def process_exception(self, request, exception):
             response = None
 
         return response
+
+
+class MobileDetectMiddleware:
+    """Mobile Detect Middleware
+
+    Redirects mobile devices to `/mobile` path if `early_access_code` is in the
+    URL. The Deep Linking will catch the URL and it will be opened in the app.
```

Comment:
This docstring also needs to be updated after the `/mobile` part is made dynamic.